### PR TITLE
Update routes-and-templates.md

### PR DESCRIPTION
**Pull request by ankitsinghaniyaz**
ember 2.6 does not create the application.hbs template automatically as suggested in the guide. I have modified it at places to match this new step.
1. Added command to generate the application template
2. modified its references at several places.

### CODE REVIEW

File: source/localizable/tutorial/routes-and-templates.md [hunk 1]
```diff
@@ -218,14 +224,12 @@ Let's update our `index.hbs` with some HTML for our home page and our links to t
 
 In addition to providing button-style links in each route of our application, we would like to provide a common banner to display both the title of our application, as well as its main pages.
 
-When you create an Ember application with Ember CLI as we did, it generates a template called `application.hbs`.
+We generated a template called `application.hbs`.
```
Comment (yawboakye) [inline source/localizable/tutorial/routes-and-templates.md]: @ankitsinghaniyaz This part doesn't read naturally to me.

Comment (ankitsinghaniyaz) [inline source/localizable/tutorial/routes-and-templates.md]: @yawboakye Would you like me to rephrase the sentence?

**Comment by toddjordan** [comment]
Already a pr for this here: https://github.com/emberjs/guides/pull/1533
We'll need to reconcile these.

File: source/localizable/tutorial/routes-and-templates.md (hunk 1 above)
Comment (yawboakye) [inline source/localizable/tutorial/routes-and-templates.md]: Please see if there's a way to reconcile with https://github.com/emberjs/guides/pull/1533. I think @toddjordan deleted that whole part.

**Comment by toddjordan** [comment]
Going to close this in favor or #1533.  Thanks again for the contribution though, you efforts are appreciated!

**Pull request closed without merging**